framework/source/class/qx/bom/element/Decoration.js

```diff
@@ -131,11 +131,45 @@ qx.Class.define("qx.bom.element.Decoration",
       var tag = this.getTagName(repeat, source);
       var ret = this.getAttributes(source, repeat, style);
       var css = qx.bom.element.Style.compile(ret.style);
+      var isFont = source.startsWith("@");
+
+      if (isFont) {
+        var ResourceManager = qx.util.ResourceManager.getInstance();
+        var font = qx.theme.manager.Font.getInstance().resolve(source.match(/@([^/]+)/)[1]);
+
+        var styles = font.getStyles();
```

Comment:
The font styles map must be cloned otherwise the original map would be manipulated